solvers/spams.py

```diff
@@ -0,0 +1,40 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    from spams import lasso, fistaFlat
+    import numpy as np
```

Comment:
Numpy is a deps of `benchopt` so you can import it outside of the `safe_import_context`.

Reply:
done